app/views/comfy/admin/non_primitive_properties/_nested_form.haml

```diff
@@ -12,12 +12,41 @@
   .form-group{style: "#{'display:none' unless resource.field_type == 'richtext'}", id: "rich_text_field_#{index}"}
     = label_tag 'Default Content'
     = rich_text_area_tag "api_namespace[non_primitive_properties_attributes][#{index}][content]", resource.content
+  .form-group.toggle-file-attachment.align-items-center{style: "#{resource.field_type == 'richtext' ? 'display:flex' : 'display:none'}", id: "rich_text_field_allow_attachments_#{index}"}
+    = label_tag 'Allow Attachments', nil, class: 'm-0 mr-1'
+    -# workaround reference: https://github.com/rails/rails/issues/5937, https://stackoverflow.com/a/20380051
```

Comment:
@alis-khadka this link results in a 404: https://github.com/rails/rails/issues/5937,%20https://stackoverflow.com/a/20380051

Reply:
Well, the link is good on my end.
![image](https://user-images.githubusercontent.com/25191509/177008654-2e7914a1-6eb1-46e3-8767-93bbe60698f3.png)

Reply:
ah its working now, thanks for the screenshot though @alis-khadka